dev-client/src/screens/ProjectTeamScreen/components/UserList.tsx

```diff
@@ -15,165 +15,48 @@
-  Pick<ItemProps, 'memberAction'> & {
-    isCurrentUser: boolean;
-    currentUserRole: ProjectRole;
-  }) =>
-  !isCurrentUser && currentUserRole === 'MANAGER' ? (
-    <Pressable onPress={memberAction}>{children}</Pressable>
-  ) : (
-    <>{children}</>
-  );
-
-function UserItem({
-  membership,
-  user,
-  currentUserId,
-  removeUser,
-  memberAction,
-  currentUserRole,
-}: ItemProps) {
-  const {t} = useTranslation();
-  const isCurrentUser = useMemo(() => {
-    return user.id === currentUserId;
-  }, [user, currentUserId]);
-
-  const userName = useMemo(() => {
-    let name = formatName(user.firstName, user.lastName);
-
-    if (isCurrentUser) {
-      name += ` (${t('general.you')})`;
-    }
-    return name;
-  }, [user, isCurrentUser, t]);
-
-  return (
-    <Box borderBottomWidth="1" width={275} py={2}>
-      <VStack>
-        <UserWrapper
-          currentUserRole={currentUserRole}
-          isCurrentUser={isCurrentUser}
-          memberAction={memberAction}>
-          <HStack space={3} justifyContent="space-between" alignItems="center">
-            <Box>
-              <Image
-                variant="profilePic"
-                source={{uri: user.profileImage}}
-                alt="profile pic"
-              />
-            </Box>
-            <Text flex={3}>{userName}</Text>
-            <Box>
-              <Badge
-                variant="chip"
-                bg="primary.lighter"
-                py="5px"
-                px="10px"
-                _text={{color: 'text.primary'}}>
-                {t('general.role.' + membership.userRole)}
-              </Badge>
-            </Box>
-          </HStack>
-        </UserWrapper>
-        {isCurrentUser && (
-          <ConfirmModal
-            trigger={onOpen => (
-              <LeaveProjectTrigger
-                onOpen={onOpen}
-                message={t('projects.team.leave_project_modal.trigger')}
-              />
-            )}
-            title={t('projects.team.leave_project_modal.title')}
-            body={t('projects.team.leave_project_modal.body')}
-            actionName={t('projects.team.leave_project_modal.action_name')}
-            handleConfirm={removeUser}
-          />
-        )}
-      </VStack>
-    </Box>
-  );
-}
-
 export const UserList = ({
   memberships,
-  userAction,
+  removeUser,
   currentUserId,
   memberAction,
   currentUserRole,
 }: ListProps) => {
+  /* (The number of managers in a project affects whether certain screen controls are available) */
+  const projectHasSingleManager = useMemo(
+    () => memberships.filter(m => m[0].userRole === 'MANAGER').length === 1,
+    [memberships],
+  );
+
   return (
     <FlatList
       data={memberships}
       renderItem={({item: [membership, user]}) => (
         <UserItem
           membership={membership}
           user={user}
-          currentUserId={currentUserId}
-          removeUser={userAction(membership)}
+          isForCurrentUser={user.id === currentUserId}
```

Comment:
@paulschreiber @knipec I've renamed some parameters. My reasoning is as follows:

- "current user" is well-established terminology in the mobile codebase for "the user who is currently using the application." However, I have tried to indicate with this new name that the parameter controls whether the user item is for the "current user", as opposed to another user. I do not feel comfortable introducing new one-off terminology here to replace "current user", though.
- "is manager view" indicates whether the component is being shown in a manager's view of the application, rather than the prior name which was ambiguous as to who the "manager" in question was.
- I have changed the name of the variable indicating whether the project has only one manager to try to convey that it is part of the context in which the component is being shown.